specification/communication/data-plane/Email/preview/2021-10-01-preview/examples/SendEmail.json

```diff
@@ -2,21 +2,16 @@
   "parameters": {
     "endpoint": "https://contoso.westus.communications.azure.com",
     "api-version": "2021-10-01-preview",
+    "x-ms-client-request-id": "1377af77-47e8-4fb3-8e84-5a848305daea",
     "repeatability-request-id": "8540c0de-899f-5cce-acb5-3ec493af3800",
     "repeatability-first-sent": "Sun, 06 Nov 1994 08:49:37 GMT",
     "authorization": "HMAC-SHA256 SignedHeaders=x-ms-date;host;x-ms-content-sha256&Signature=aaaa",
-    "emailMessage": {
-      "headers": [
-        {
-          "name": "ClientCorrelationId",
-          "value": "123"
-        },
-        {
-          "name": "ClientCustomHeaderName",
-          "value": "ClientCustomHeaderValue"
-        }
-      ],
-      "sender": "abc@domain.com",
+    "message": {
+      "headers": {
+        "ClientCorrelationId": "123",
+        "ClientCustomHeaderName": "ClientCustomHeaderValue"
```

Comment:
What are these two headers?

Reply:
These headers are meant to be any custom email headers that the clients want to send through and receive on the other end (for example, any kind of headers for tracking, debugging, etc.). Currently, we allow any header name/value pair here.

Reply:
No action here.